ethcore/src/service.rs

```diff
@@ -151,6 +154,14 @@ impl ClientService {
 	pub fn add_notify(&self, notify: Arc<ChainNotify>) {
 		self.client.add_notify(notify);
 	}
+
+	/// Start STRATUM job dispatcher and register it in the miner
+	pub fn register_stratum(cfg: &StratumOptions, miner: Arc<Miner>, client: Weak<Client>) -> Result<(), StratumError> {
```

Comment:
What's this function's relation to the `ClientService`? It doesn't initialize a `Service` or take one as an argument. Not convinced this belongs here.